.github/workflows/deploy-ghpages-latest-stable.yml

```diff
@@ -37,15 +37,13 @@ concurrency:
 jobs:
   check-label:
     runs-on: ubuntu-latest
-    outputs:
-      label: ${{ steps.valid_label.outputs.value}}
     steps:
       - name: checks for the label 
         id: valid_label
         run: |
           if [[ "${{ inputs.trigger-label }}" == "latest" ]] || [[ "${{ inputs.trigger-label  }}" == "stable" ]];
             then
-                echo "value=true" >> $GITHUB_OUTPUT
+                exit 0
```

Comment:
This is the default, you do not need to make it explicit.
Just add a negation to the if, and keep only the current `else` branch.